sdk/storage/tests.yml

```diff
@@ -1,30 +1,24 @@
 trigger: none
 
 extends:
-  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: storage
     BuildInParallel: true
     TimeoutInMinutes: 180
-    Location: canadacentral
     CloudConfig:
       Preview:
-        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
-      PrivatePreview:
+        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview-test)
+        Location: 'canadacentral'
+      UsGov:
+        SubscriptionConfiguration: $(sub-config-gov-test-resources-test)
+      China:
+        SubscriptionConfiguration: $(sub-config-cn-test-resources-test)
+      PrivatePreview:
         SubscriptionConfiguration: $(sub-config-storage-test-resources)
-    Clouds: Preview
+    Clouds: 'Preview,UsGov,China'
```

Comment:
We should not be running `China` and `UsGov` in the nightly runs, only the weekly runs.  Please discuss with @benbp if confirmation is needed.

Leaving one comment, but this applies to all services.